logging-api/src/main/java/com/cisco/oss/foundation/logging/transactions/HttpLogger.java

```diff
@@ -189,7 +189,7 @@ protected void addVerbosePropertiesStart(final HttpServletRequest request, Strin
             if (requestHeaders != null) {
                 this.properties.put(HttpVerbosePropertyKey.RequestHeaders.name(), requestHeaders);
             }
-            if (requestBody != null) {
+            if (requestBody != null && !ConfigurationUtil.INSTANCE.allowHeadersAndResponseBodyOnly()) {
```

Comment:
See if (requestBody != null && ConfigurationUtil.INSTANCE.allowHeadersAndRequestBodyOnly()) is more appropriate.Otherwise, for not to dump request body, we need to set responseBodyOnly flag to  true, which is bit confusing IMO. Check the the same in other places( Line 232 and 245)